arangod/Aql/QueryString.h

```diff
@@ -65,7 +65,7 @@ class QueryString {
   std::string extractRegion(int line, int column) const;
 
  private:
-  std::string const _queryString;
+  std::string _queryString;
```

Comment:
why is this not const anymore ?

Reply:
because of the class' default assignment operator. this disallows const members, and new clang versions warn about this.